src/LegendComponent/LegendComponent.js

```diff
@@ -87,7 +87,7 @@ class LegendComponent {
     if (!this.config.legendEntryMaxLength) {
       return 0;
     }
-    LabelUtil.handleLabelWrap(leg, ' g > text.legend-title', 25, 1.2, true, this.config.legendEntryMaxLength);
+    LabelUtil.handleLabelWrap(leg, ' g > text.legend-title', 25, 1.2, true, 20);
```

Comment:
This looks odd. Don't we want this configurable?

Reply:
We may want that to be configurable in the future, but the config option `legendEntryMaxLength` was actually misplaced here as the `distanceBuffer` parameter influences something else entirely.